tests/control/conftest.py

```diff
@@ -0,0 +1,18 @@
+import json
+from pathlib import Path
+
+from pytest import fixture
+
+FIXTURES_DIR = Path(__file__).resolve().parent / "fixtures"
+
+
+@fixture(scope="session")
+def preview_mapping():
```

Comment:
Using the `session` scope means the data will be holded in memory till the end of the tests. Maybe `module` is better ?

Reply:
Right 👍